### DISCUSSION TITLE
Why does the 13.5kWH Powerwall2 only appear as 8kWH in Powerwall-Dashboard?

### DISCUSSION THREAD

**Discussion by davehatton** [jasonacox/Powerwall-Dashboard#243] [Q&A] [answered]
Hi all,

Firstly, I'm a new Solar Panel / Powerwall2 owner, it has only been running for a week, and I want to say how much I appreciate the work done on Powerwall-Dashboard ... it is great.  

Now my question that I can't find an answer for, maybe it is obvious and I'm missing something?  

I understand the Powerwall2 has a capacity of 13.5kWH.  

So far Powerwall-Dashboard shows my PW2 fully charged (100%) at 8kWH.  
I see this on also on the examples pictures for this project and in screen shots in questions.  

Why is it not 13.5kWH please? What am I missing?

**Comment by jgleigh** [comment]
Can you post a picture of what you're seeing?

**Comment by davehatton** [comment]
So I think I may have an answer to some extent for myself

![Screenshot from 2023-04-12 15-10-45](https://user-images.githubusercontent.com/49393/231484902-a6589146-0c32-4c08-a193-59aab1d46f44.png)

In that picture, the y-axis only goes to 8kWH but now thinking about it, there is also 100%, so am I correct in thinking that the 100% is related to the red battery charge line and the 8kW is releated to house usage and solar generation.

It isn;t saying that 100% battery is at 8kW


Then I see on a lower graph by battery capacity

![Screenshot from 2023-04-12 15-11-15](https://user-images.githubusercontent.com/49393/231486034-27d8f015-7eee-4cb4-b67a-6d3a3418b8b8.png)

THis suggests my 13.5kWH PW2 has a 15kWH charge?

**Reply by jgleigh** [reply, marked as answer]
The top graph is instantaneous power, expressed in kW. The bottom graph is battery capacity expressed in kWh. One kW used for an hour equals one kWh.

**Reply by richardklam** [reply]
In addition, everything on the Top graph is scaled to kW and most of the units on the top graph is express in Power to the left. The SOC of the Powerwall is on the right side of the Top graph with units 100% to -100%. for Charge, Clouds, and Reserve parameters.

**Comment by davehatton** [comment]
Thank you for the replies, that cleared it up.
